libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/task/OfflineRouteRetrievalTask.kt

```diff
@@ -0,0 +1,66 @@
+package com.mapbox.navigation.route.onboard.task
+
+import android.os.AsyncTask
+import com.google.gson.Gson
+import com.mapbox.navigation.base.logger.model.Message
+import com.mapbox.navigation.base.logger.model.Tag
+import com.mapbox.navigation.base.route.dto.RouteResponseDto
+import com.mapbox.navigation.base.route.dto.mapToModel
+import com.mapbox.navigation.base.route.model.Route
+import com.mapbox.navigation.logger.MapboxLogger
+import com.mapbox.navigation.navigator.MapboxNativeNavigator
+import com.mapbox.navigation.route.onboard.OnOfflineRouteFoundCallback
+import com.mapbox.navigation.route.onboard.model.OfflineError
+import com.mapbox.navigation.route.onboard.model.OfflineRouteError
+import com.mapbox.navigator.RouterResult
+
+internal class OfflineRouteRetrievalTask(
+    private val navigator: MapboxNativeNavigator,
+    private val callback: OnOfflineRouteFoundCallback
+) : AsyncTask<String, Void, List<Route>>() {
+
+    @Volatile
+    private lateinit var routerResult: RouterResult
+
+    private val logger = MapboxLogger
```

Comment:
Do we need logger instance in this class? I guess we can use MapboxLogger explicitly here, like a MapboxLogger.e(...), wdyt?

Reply:
We shouldn't use the `liblogger` in this module at all, because it's unreleased and will not compile downstream.

If we need to log, the router should accept the logger as a parameter and use only the logger interface from `base`.